test-cases/longevity/longevity-multi-dc-rack-aware-with-znode-in-diff_dc.yaml

```diff
@@ -7,7 +7,7 @@ stress_cmd: ["cassandra-stress write cl=LOCAL_QUORUM duration=720m -schema 'repl
              "cassandra-stress read  cl=LOCAL_QUORUM duration=720m -schema 'replication(strategy=NetworkTopologyStrategy,replication_factor=3,eu-northscylla_node_north=0) compaction(strategy=SizeTieredCompactionStrategy)' -port jmx=6868 -mode cql3 native -rate threads=40 -pop 'dist=uniform(1..20971520)' -col 'n=FIXED(10) size=FIXED(512)' -log interval=5 -errors retries=50",
              ]
 
-n_db_nodes: '3 3 0'
+n_db_nodes: '4 4 0'
```

Comment:
why it's being increased ? (the commit message doesn't tell why)

Reply:
Added comment to PR message. Added it to safely run decommission with tablets